src/jsx/repo.jsx

```diff
@@ -292,3 +301,26 @@ var Repo = React.createClass({
     }
   }
 });
+
```

Comment:
Making this change happily. To wax intellectual, I disagree about lodash vs. polyfills. Soon all of these methods will be available cross-browser and getting into the habit of using the built-in JavaScript methods seems more forward thinking.